frontend/js/store/index.js

```diff
@@ -1,20 +1,14 @@
-import { routerMiddleware } from 'connected-react-router';
-import { createBrowserHistory } from 'history';
```

Comment:
these libraries aren't recommended by the react-router team. I don't see the benefit as well so I removed them. see https://v5.reactrouter.com/web/guides/deep-redux-integration